agent/exporter/src/main/java/com/microsoft/applicationinsights/agent/Exporter.java

```diff
@@ -424,18 +432,20 @@ private static void applyHttpRequestSpan(Map<String, AttributeValue> attributes,
     }
 
     private static void applyDatabaseQuerySpan(Map<String, AttributeValue> attributes, RemoteDependencyTelemetry telemetry, String component) {
-        String type = removeAttributeString(attributes, "db.type");
-        if ("sql".equals(type)) {
-            type = "SQL";
+
+        String type = removeAttributeString(attributes, SemanticAttributes.DB_SYSTEM.key());
+
+        if ("mssql".equals(type)) {
+            type = "mssql";
         }
```

Comment:
this doesn't look needed
```suggestion
```

Reply:
It used to be SQL.  I can remove it and ensure jdbc still passes.

Reply:
Oh yes, let's map all of the known SQL-based `db.system` values to `SQL`: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/database.md#notes-and-well-known-identifiers-for-dbsystem

* db2
* derby
* mariadb
* mssql
* mysql
* oracle
* postgresql
* sqlite
* other_sql